charts/osdfir-infrastructure/Chart.yaml

```diff
@@ -1,24 +1,29 @@
 apiVersion: v2
 name: osdfir-infrastructure
-version: 0.3.3
+version: 0.3.4
 description: A Helm chart for Open Source Digital Forensics Kubernetes deployments.
 keywords:
 - timesketch
 - turbinia
+- yeti
 - dfir
 - analysis
 - processing
 - cloud
 home: "https://github.com/google/osdfir-infrastructure"
 dependencies:
-- condition: timesketch.enabled
+- condition: global.timesketch.enabled
   name: timesketch
   repository: https://google.github.io/osdfir-infrastructure/
-  version: 0.3.4
-- condition: turbinia.enabled
+  version: 0.3.5
+- condition: global.turbinia.enabled
   name: turbinia
   repository: https://google.github.io/osdfir-infrastructure/
   version: 0.3.3
+- condition: global.yeti.enabled
+  name: yeti
+  repository: https://google.github.io/osdfir-infrastructure/
+  version: 0.1.0
```

Comment:
This is the version number of the chart, not Yeti, right?

Reply:
Yep exactly, although let me bump that to 1.0.0 (going to do that with the rest of the charts as well) given that we have adequate testing in place now

Reply:
Actually going to do in separate PR along with the other charts